rpcs3/Emu/RSX/Common/texture_cache.h

```diff
@@ -1022,7 +1022,7 @@ namespace rsx
 				else
 				{
 					// This is a read and all overlapping sections were RO and were excluded (except for cause == superseded_by_fbo)
-					AUDIT(cause.skip_fbos() || (cause.is_read() && !result.sections_to_exclude.empty()));
+					// AUDIT(cause.skip_fbos() || (cause.is_read() && !result.sections_to_exclude.empty()));
```

Comment:
I'd rather fix the audits than silently removing them. What's wrong with them as-is? And why do they not break when compiling with gcc on other platforms?

Reply:
Reverted. It may be because noone is using a debug build and the audits only work in debug mode.